Makefile

```diff
@@ -375,6 +411,13 @@ internal-clean: $$(KIND)
 internal-distclean: $$(KIND)
 	@rm -rf ./_build/$(KIND)
 
+prod-deb-package:
+	export DEBEMAIL=$(AE_DEB_EMAIL); \
+	export DEBFULLNAME=$(AE_DEB_NAME) ;\
```

Comment:
How does quoting / splitting work here? Could you please review other vars in this recipe too please?

Reply:
Your suggestion is to quote AE_DEB_NAME and the rest?

```
export DEBEMAIL="$(AE_DEB_EMAIL)"
```
I am not sure if this will affect something. Working on CI integration in a separate branch shows it behaves as expected - no errors were produced because of spaces or new lines.

Reply:
I see AE_DEB_NAME has quotes already `"Aeternity Team"` so maybe those are kept. Ok.

Reply:
I can confirm it works.